test/functional/Java8andUp/playlist.xml

```diff
@@ -405,6 +405,26 @@
 	</test>
 	<!-- Attach API tests -->
 
+	<test>
+		<testCaseName>TestJps</testCaseName>
+		<command>$(JAVA_COMMAND) $(JVM_OPTIONS) \
+	-cp $(Q)$(RESOURCES_DIR)$(P)$(TESTNG)$(P)$(TEST_RESROOT)$(D)GeneralTest.jar$(Q) \
+	-Dcom.ibm.tools.attach.enable=yes \
+	-Djdk.attach.allowAttachSelf=true \
+	-Dcom.ibm.tools.attach.timeout=15000 \
+	org.testng.TestNG -d $(REPORTDIR) $(Q)$(TEST_RESROOT)$(D)testng.xml$(Q) -testnames TestJps \
+	-groups $(TEST_GROUP) \
+	-excludegroups $(DEFAULT_EXCLUDE); \
+	$(TEST_STATUS)</command>
+		<levels>
+			<level>sanity</level>
+			<level>extended</level>
```

Comment:
- Should it be either sanity or extended?
- Please add impls

Reply:
> * Should it be either sanity or extended?

Both.  I annotated one test as "level.sanity" and the more extensive tests "level.extended".

> * Please add impls

Done

Reply:
Please separate into to distinct targets if you want some in sanity and some in extended.  Highly encourage keeping them together in extended.

Reply:
Thanks @smlambert .  All tests moved to extended.